.dockerignore

```diff
@@ -1,4 +1,4 @@
-packages
+packages/sdk
```

Comment:
Maybe it would be better to still ignore the whole "packages" folder, and only unignore the "connector-types" package.

Reply:
true.